utils/generate_types/overrides-test.d.ts

```diff
@@ -208,11 +208,12 @@ export interface PlaywrightWorkerOptions {
   channel: BrowserChannel | undefined;
   launchOptions: LaunchOptions;
   connectOptions: ConnectOptions | undefined;
-  screenshot: 'off' | 'on' | 'only-on-failure';
+  screenshot: ScreenshotMode | { mode: ScreenshotMode } & Pick<PageScreenshotOptions, 'fullPage' | 'omitBackground'>;
```

Comment:
In addition to this change, please also update the [api docs here](https://github.com/microsoft/playwright/blob/main/docs/src/test-api/class-testoptions.md?plain=1#L191-L200). Take a look at [the similar `trace` option here](https://github.com/microsoft/playwright/blob/main/docs/src/test-api/class-testoptions.md?plain=1#L221-L237) for an example.

Reply:
Thanks! Added